kolibri/deployment/default/custom_django_cache.py

```diff
@@ -23,12 +47,17 @@ def add(
         tag=None,
         retry=True,
     ):
-        try:
-            return super(CustomDjangoCache, self).add(
-                key, value, timeout, version, read, tag, retry
-            )
-        except sqlite3.OperationalError:
-            return False
+        return self.try_execute(
+            method_name="add",
```

Comment:
Sorry, just one more thing - I think this probably does work to pass positional arguments as named keyword arguments, but I would find it less confusing if `method_name` and `error_return_value` were pass just as position arguments in each instance here, so:
```
return self.try_execute(
    "add",
    False,
```

And so on.

Reply:
sure, sorry for the trouble